salt/utils/nacl.py

```diff
@@ -199,12 +199,7 @@ def enc(data, **kwargs):
         )
         kwargs['sk'] = kwargs['key']
 
-    # ensure data is in bytes
-    data = salt.utils.stringutils.to_bytes(data)
-
     box_type = _get_config(**kwargs)['box_type']
-    if box_type == 'sealedbox':
```

Comment:
Why is this being removed?

Reply:
It runs `sealedbox_encrypt` by default. I didn't see the need for an additional check.

Reply:
Ah. Gotcha.